### ISSUE TITLE
docker easy start instructions fail with the make command.

### ISSUE THREAD

**Issue by rluijk** [trailofbits/manticore#1189]
### OS / Environment

No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.5 LTS
Release:	16.04
Codename:	xenial

### Manticore version

Metadata-Version: 1.0
Version: 0.2.1.1
### Python version

Python 2.7.12

### Dependencies

capstone==3.0.5.post1
future==0.16.0
manticore==0.2.1.1
ply==3.11
pyelftools==0.25
pyevmasm==0.1.1
pysha3==1.0.2
PyYAML==3.13
unicorn==1.0.1
z3-solver==4.8.0.0.post1

### Summary of the problem
Docker version as described in the quick start guide gives error when giving the "make" command. 
Looks like it is using a python 2 instead of 3 version?

### Step to reproduce the behavior
Follow steps as described in the quick start guide on the github starting page for docker 

### Expected behavior
working version

### Actual behavior
make command stuck, with calling python 2 version?

### Any relevant logs
```
rluijks-MBP:manticore rluijk$ docker run -it -v $PWD/examples:/home/manticore/examples trailofbits/manticore
manticore@018c73b96861:~$ cd examples/linux/
manticore@018c73b96861:~/examples/linux$ make
gcc -O3 -static basic.c -o basic
gcc -O3 -static sindex.c -o sindex
sindex.c: In function 'main':
sindex.c:72:9: warning: implicit declaration of function 'read' [-Wimplicit-function-declaration]
     if (read(0, &cmd, sizeof(cmd)) != sizeof(cmd))
         ^
gcc -O3 -static strncmp.c -o strncmp
strncmp.c: In function 'main':
strncmp.c:59:5: warning: implicit declaration of function 'read' [-Wimplicit-function-declaration]
     read(0, buffer, 0x100);
     ^
strncmp.c:61:9: warning: implicit declaration of function 'strcmp' [-Wimplicit-function-declaration]
     if (strcmp(buffer, "ZARAZA") == 0 )
         ^
gcc -O3 -static arguments.c -o arguments
gcc -O3 -static ibranch.c -o ibranch
gcc -O3 -static sendmail.c -o sendmail
sendmail.c: In function 'main':
sendmail.c:63:1: warning: implicit declaration of function 'read' [-Wimplicit-function-declaration]
 read(0,buffer,200);
 ^
python3 crackme.py > crackme.c
Traceback (most recent call last):
  File "crackme.py", line 154, in <module>
    print(PROGRAM)
UnicodeEncodeError: 'ascii' codec can't encode character '\ufffd' in position 1938: ordinal not in range(128)
Makefile:33: recipe for target 'crackme.c' failed
make: *** [crackme.c] Error 1
manticore@018c73b96861:~/examples/linux$ manticore basic
Traceback (most recent call last):
  File "/usr/local/bin/manticore", line 9, in <module>
    load_entry_point('manticore==0.2.1.1', 'console_scripts', 'manticore')()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 542, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2569, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2229, in load
    return self.resolve()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2235, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python2.7/dist-packages/manticore/__init__.py", line 1, in <module>
    from .manticore import Manticore  # noqa
  File "/usr/local/lib/python2.7/dist-packages/manticore/manticore.py", line 21, in <module>
    from .core.executor import Executor
  File "/usr/local/lib/python2.7/dist-packages/manticore/core/executor.py", line 9, in <module>
    from ..utils.event import Eventful
  File "/usr/local/lib/python2.7/dist-packages/manticore/utils/event.py", line 38
    class Eventful(object, metaclass=EventsGatherMetaclass):
                                    ^
SyntaxError: invalid syntax
manticore@018c73b96861:~/examples/linux$ python
Python 2.7.12 (default, Dec  4 2017, 14:50:18)
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> exit()
manticore@018c73b96861:~/examples/linux$ python3
Python 3.5.2 (default, Nov 23 2017, 16:37:01)
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

**Comment by akhavr** [comment]
It won't start even after changed to `python3`:

```
manticore@7f719747b304:~/examples/linux$ head -2 `which manticore`
#!/usr/bin/python3
# EASY-INSTALL-ENTRY-SCRIPT: 'manticore==0.2.1.1','console_scripts','manticore'
manticore@7f719747b304:~/examples/linux$ manticore basic
Traceback (most recent call last):
  File "/usr/local/bin/manticore", line 5, in <module>
    from pkg_resources import load_entry_point
ImportError: No module named 'pkg_resources'
```

**Comment by disconnect3d** [comment]
Closing this as this is not anymore the case.

The instructions will also be updated a bit by https://github.com/trailofbits/manticore/pull/1266 and we are going to have docker image tags for each Manticore release. I've already pushed 0.2.2: https://hub.docker.com/r/trailofbits/manticore/tags/ but all the others will be built automatically. Also, the `latest` will always point to last release and `master` to this repository's `master` branch.
